src/themes/base/components/Indicator/indicatorStyles.ts

```diff
@@ -0,0 +1,15 @@
+import * as _ from 'lodash'
+import { ComponentStyleFunctionParam, ICSSInJSStyle } from '../../../types'
+import { IndicatorVariables } from './indicatorVariables'
+import { IndicatorProps } from '@stardust-ui/react'
+
+export default {
+  root: ({
+    props: { color },
```

Comment:
Do we want an indicator to be able to specify it's own color? I think the indicator color should inherit from the current color by default. Where the current color would depend on the color set by the containing components (ie Menu, Accordion in these cases)

Reply:
Agree, will remove the color prop from this one. Thanks!